paws.common/R/config.R

```diff
@@ -182,14 +182,41 @@ get_instance_metadata <- function(query_path = "") {
   if (trimws(tolower(get_env("AWS_EC2_METADATA_DISABLED"))) %in% c("true", "1")) {
     return(NULL)
   }
+  # Get token timeout for IMDSv2 tokens
+  tokentimeout=trimws(tolower(get_env("PAWS_EC2_IMDSV2_TOKEN_TIMEOUT_SECONDS")))
+  if (is.na(as.numeric(tokentimeout))) {
+        tokentimeout="30"
+  }
+  token <-  "" # Token to be used in case of more secure IMDSv2 authentication
+  #try IMDSv2  (more information): https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-instance-metadata-service.html
+   metadata_token_url <-  file.path(
+    "http://169.254.169.254/latest/api/token"
+  )
```

Comment:
small thing, do we need file.path for just assigning a character :)

Reply:
This was taken over from the previous function - I just kept it the same to avoid as little inconsistencies as possible.